src/reducers/index.js

```diff
@@ -21,17 +22,16 @@ const initialStore = {
   community_posts: [],
   diaryData: [],
   users: [],
-  foods_meals_users: [],
   isLoggedIn: false,
+  addFood: {},
```

Comment:
maybe change variable for clearer naming convention.

consider: 'addedFood' or 'submittedFood'